### ISSUE TITLE
Tutorial for adding systemd-shim?

### ISSUE THREAD

**Issue by sunbinyuan** [DesktopECHO/linuxdeploy-images#2]
Hi there, was wondering if you had any tutorial to install and patch systemd-shim into any linux distro to be ran on Linux Deploy? Was this the shim package that you used by any chance https://github.com/MX-Linux/systemd-shim? Thanks in advance!

**Comment by DesktopECHO** [comment]
Hello, yes that is the one. 
It's available as a DEB package as well, so you can try installing that way to see if it's suitable for your purposes.

**Comment by sunbinyuan** [comment]
Awesome, thanks so much for the quick reply! One other question about the NextCloudPi port that you made for Android. I noticed on the Readme that it says that it work on armv8 devices but I found some seemingly armv7 specific logic here https://github.com/DesktopECHO/nextcloudpi/blob/512e38084a0b71e283776a97d265046f0ffbc2bd/build/linuxdeploy/notify_push. Is that intended to also work on armv8 also or would I need to change the folder to reflect the correct architecture?
